lib/logic/common/plan_instance_cubit.dart

```diff
@@ -20,22 +20,22 @@ class PlanInstanceCubit extends ReloadableCubit {
 	final DataRepository _dataRepository = GetIt.I<DataRepository>();
 	final TaskInstanceService _taskInstancesService = GetIt.I<TaskInstanceService>();
 	final UIDataAggregator _dataAggregator = GetIt.I<UIDataAggregator>();
-	final ObjectId _planInstanceId;
+	UIPlanInstance uiPlanInstance;
 
 	PlanInstance planInstance;
 
-	PlanInstanceCubit(this._planInstanceId, ModalRoute modalRoute) : super(modalRoute);
+	PlanInstanceCubit(this.uiPlanInstance, ModalRoute modalRoute) : super(modalRoute);
```

Comment:
In this case it's going to be a little more tricky as I have not yet written any support for custom initial states in reloadable cubits. I will look into this tomorrow

Reply:
I will leave that case for when I will be properly implementing the page model passing functionality as I see no elegant way of doing it temporarily right now. For now you can leave this cubit as-is.